lib/req-res/status.js

```diff
@@ -61,7 +61,7 @@ module.exports = function ( options ) {
 						'data'        : mixins.cloneDeep( errorData ) || {},
 						'code'        : code,
 						'messageType' : options.messageType,
-						'message'     : mixins.cloneDeep( options.message ) || {},
+						'msg'         : mixins.cloneDeep( options.message ) || {},
```

Comment:
Sorry I'm an idiot. You have `errorMsg` right below this. That's what I was thinking.